content/lesson-development.md

```diff
@@ -4,16 +4,98 @@
+single **learning objective**: applying the forking workflow to interact
+with open source codes. The second half branches of to multiple useful
+directions.
+
+ * **Days 1-2**: [Introduction to version control](https://coderefinery.github.io/git-intro/)
+ * **Day 3**: [Collaborative distributed version control](https://coderefinery.github.io/git-collaborative/)
+ * **Day 4**: [Reproducible research](https://coderefinery.github.io/reproducible-research/)
+ * **Day 4**: [Social coding and open software](https://coderefinery.github.io/social-coding/)
+ * **Day 5**: [How to document your research software](https://coderefinery.github.io/documentation/)
+ * **Day 5**: [Jupyter notebooks](https://coderefinery.github.io/jupyter/) 
+ * **Day 6**: [Automated testing](https://coderefinery.github.io/testing/)
+ * **Day 6**: [Modular code development type-along](https://coderefinery.github.io/modular-type-along/)
+
+
+
+## Other lessons
+
+
+# Backward lesson design (15min)
+
+### Why "Backward"
+
+The "natural" way to design a lesson, that people often default to, is
+to write down material you want to cover and then springle in related
+exercises. This approach has a couple of significant downsides:
+ - The content feel meandering and it's unclear what is meant to be learned.
+ - The exercises and exam may not test the intended learning outcomes.
+
+In backward lesson design we start from the learning objective and the
+"final exam" and work backwards from there to 
+
+### The method:
+
+ 1. Understand you learners
+ 2. brainstorm rough ideas
+ 3. Start from learning objectives. Write down 1-3 overarching learning
+   objectives for the lesson.
+    - think of the things your learners will
+      be able to do at the end of the lesson. Think simple! The simpler
+      the better. Think of three main points they will remember, of which
+      maybe one or two are a concrete skill.
+ 4. Write an summative assesment for each outcome. Try to think of engaging
+    and active exercises.
+ 5. For a longer lesson, add intermediate goals and formative assesments
+    to bridge between the starting point and the learning objective and put them
+    in a reasonable order.
+ 6. Write just enough material to get from one assessment (exercise) to another.
+ 7. Describe the course so the learners know if it is relevant to them.
+
+
+# Contributing issues and pull requests (5min)
+
+
+# Exercises:
+
+## Lesson design exercise (10-20min, or longer)
+
+ - Pick a lesson topic. Any of the Tools Workshop lessons are OK,
+   or you can choose any other topic you might want to teach.
+ - Write down three learning objectives. What abilities would you
+   expect the student to have at the end of the lesson?
+ - Brainstorm an exercise that demonstrates one of those objectives.
+ - If you have time:
+   - Create two more exercises
+   - Write chapter headers for a quick structure
+
+
+## Contributing exercies (10-20min)
```

Comment:
Uh yea, I like that! 
Let's make sure we have some nice issues there that we can suggest in collaborative doc :)